proxy/logMonitor_test.go

```
@@ -21,27 +21,34 @@ func TestLogMonitor(t *testing.T) {
 	client2Messages := make([]byte, 0)
 
 	var wg sync.WaitGroup
-	wg.Add(1)
+	wg.Add(2) // One for each client
 
+	// Write messages first
+	logMonitor.Write([]byte("1"))
```

golangci-lint (1.64.8)

Error return value of `logMonitor.Write` is not checked

(errcheck)